apps/world/src/render/render.service.ts

```diff
@@ -11,6 +11,7 @@ import { GridMapGenerator } from '../gridmap/gridmap-generator';
 import { DEFAULT_BIOMES } from '../gridmap/default-biomes';
 import { buildGridConfigs, deriveTemperature } from '../gridmap/utils';
 import { mapGridBiomeToBiomeInfo } from '../gridmap/biome-mapper';
+import { WorldUtilsService } from '../world/world-utils.service';
```

Comment:
Let's move this chunk size to a NX library so that the DM and World service can use a centralized config for things like this.